lib/ecs_rails_console/cli.rb

```diff
@@ -23,12 +27,14 @@ def run!
 
       puts "it is running on: #{public_ip}"
 
-      system("ssh -tq -oStrictHostKeyChecking=no root@#{public_ip} 'cd /app ; bin/rails console'")
+      system("ssh #{SSH_OPTIONS} #{ssh_user}@#{public_ip} 'cd /app ; #{command}'")
+    rescue Aws::ECS::Errors::ExpiredTokenException
+      puts "\nHey, it seems your token expired. Authenticate on AWS give another try."
```

Comment:
Where do they put the new credentials when they have them? In the config file?

Reply:
This gem uses AWS SDK, which stores credentials in their own config file `~/.aws/credentials`. I will make it clear in README.

Reply:
Thanks. I had assumed the config file, so I was completely wrong